tasks_modification_wizard/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Tasks modification wizard
+=========================
+
+
+This module extends the functionality of project to support ... and allow you to ...
```

Comment:
"This module extends the functionality of project and adds 
a wizard to mass editing basic information for the tasks".